### ISSUE TITLE
Ability to change the colour of an individual (or multiple) action icons after area selection.

### ISSUE THREAD

**Issue by ghost** [flameshot-org/flameshot#2016]
I'd like the ability to change the colour of my most used action buttons. For example, i find i have to hunt a little for the Copy (to clipboard) icon after making a selection. This is largely because where that icon appears varies depending on the size of the selection box.

If i could set that icon's colour to something that contrasts the rest of the icon colours, i'd be able to instantly find it - making the workflow much quicker.

For me, i'd likely have copy to clipboard, and save to disk highlighted as different colours as i use them probably 95% of the time 

Thanks for considering :)

**Comment by kpcee** [comment]
Yes, thanks mmahmoudian, that's exactly the direction I meant. :thumbsup:

**Comment by veracioux** [comment]
@mmahmoudian @borgmanJeremy I don't think the icons can be changed. Flameshot uses Qt's resource compiler (RCC), so all images are baked into the executable.

Even if you change the source SVGs before compiling flameshot, you can't really change the background color of the action buttons, because the color is taken from the `uiColor` config option.

**Comment by borgmanJeremy** [comment]
Correct I was thinking of recompiling with custom icons.
